static/src/javascripts/projects/admin/bootstraps/commercial-browser-performance.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+/* global $, Epoch*/
+import config from 'lib/config';
+import fetchJson from 'lib/fetch-json';
+import template from 'lodash/utilities/template';
+import countBy from 'lodash/collections/countBy';
+import find from 'lodash/collections/find';
```

Comment:
True. I think that's out of the scope of this PR though. I will add it to my next PR which is me typing this file.